src/transformers/chai-should.test.js

```diff
@@ -518,15 +518,23 @@ testChanged(
         expect(undefined).to.not.be.ok;
         expect(null).to.not.be.ok;
         expect(null).to.not.be.ok();
-    `,
+
+        const x = { ok: true };
+        expect(x.ok).toBeTruthy();
```

Comment:
Should this be `expect(x.ok).to.be.ok` instead of `expect(x.ok).toBeTruthy();`?

Reply:
Good question! That is actually the state of the source code after the intermediary step of running the `chai-assert.js` codemod (but before running the `chai-should.js` codemod), which I didn't articulate properly in the pull request description, sorry. I updated the test cases and pull request description for clarity but those cases can also be broken out into a separate test if you think that will make the functionality even more clear.